src/DaPigGuy/PiggyCrates/commands/CrateCommand.php

```diff
@@ -14,7 +14,14 @@
 class CrateCommand extends BaseCommand
 {
     /** @var PiggyCrates */
-    protected $plugin;
+    protected \pocketmine\plugin\Plugin $plugin;
```

Comment:
I don't think it's that bad, for clean code it's better to use `use` statements.
```php
use pocketmine\plugin\Plugin;
```
```suggestion
    protected Plugin $plugin;
```